multirpc/subpub/stream.go

```diff
@@ -10,45 +10,48 @@ import (
 )
 
 func (ps *SubPub) handleStream(stream network.Stream) {
+	// 'stream' will stay open until you close it (or the other side closes it).
+
+	log.Infof("connected to peer %s: %+v", stream.Conn().RemotePeer(), stream.Conn().RemoteMultiaddr())
+
+	// create a buffer to allow queueing up to 8 outgoing messages per peer.
```

Comment:
i left this buffer in place, although i'm not sure if it's pointless, now that writeHandler is only used for unicasts.
i mean i'm not sure how useful it is to have a buffer for unicasts.
(broadcasts do not go through this buffer anymore, they are passed directly to gossipsub)

Reply:
indeed, replaced with a bufio.Writer